@here/harp-omv-datasource/test/MapViewPickingTest.ts

```diff
@@ -391,8 +404,30 @@ describe("MapView Picking", async function() {
                           testCase.lookAt.length > 2 ? testCase.lookAt[2] : undefined
                       )
                     : rayOrigin;
+                // typeof required because the bind returns any
+                const getTile: typeof geoJsonDataSource.getTile = geoJsonDataSource.getTile.bind(
+                    geoJsonDataSource
+                );
+                if (testCase.addDependency === true) {
+                    geoJsonDataSource.getTile = function(_tileKey: TileKey, delayLoad?: boolean) {
+                        const tile = getTile(_tileKey, delayLoad);
+                        if (tile !== undefined) {
+                            tile.dependencies.push(
+                                TileKey.fromRowColumnLevel(
+                                    (_tileKey.row + 1) % _tileKey.rowCount(),
+                                    _tileKey.column,
+                                    _tileKey.level
+                                )
+                            );
+                        }
+                        return tile;
+                    };
+                }
```

Comment:
you can use sinon to simplify a little bit:
sinon.stub(geoJsonDataSource, "getTile").callsFake( yourFunction);

later to restore tje original function:
geoJsonDataSource.getTile.restore()

Reply:
Good point, will do.